src/serializer.ts

```diff
@@ -119,9 +119,11 @@ class Serializer {
    * @param {proto.FiatAmount.Currency} currency The Currency to convert.
    * @returns {String} The Currency as JSON.
    */
-  private static currencyToJSON(currency: Currency): string {
+  private static currencyToJSON(currency: Currency): string | undefined {
     switch (currency) {
-      case Currency.USD:
+      case Currency.CURRENCY_INVALID:
```

Comment:
Not sure what `CURRENCY_INVALID` will be used for, but if it should not occur (i.e. it represents an error) then we could consider throwing an error here.